frontend/src/components/Motive/motive.js

```diff
@@ -0,0 +1,56 @@
+import React from "react";
+import style from "./motive.module.css";
+
+const Motive = () => {
+  const arrayCards = [
```

Comment:
Create a new file call `dummy.js` and `arrayCard ` fetch from dummy.js

Reply:
@codewithvk I have made the required changes. Please review now.

Reply:
@vedikaag99 hey I had to add the file structure, Can you please do a re-format?

Reply:
@codewithvk hey so I'll just have to move my `Motive` folder inside of `Components` folder ,right?
Also, since `girlscript:master` now has some new commits, I can fetch upstream and update my `master` branch but how do I update my branch ie `patch-1`. Can you please guide me??

Reply:
@vedikaag99 Yes delete readme file.
also you can create new branch.